src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -30,11 +30,31 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.Display.ByGeometryColor</oldName>
-    <newName>Display.Display.ByGeometryColor</newName>
+    <newName>Modifiers.GeometryColor.ByGeometryColor</newName>
     <additionalAttributes>
       <attribute>
         <name>assembly</name>
-        <value>Display.dll</value>
+        <value>GeometryColor.dll</value>
+      </attribute>
+    </additionalAttributes>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>Display.Display.ByGeometryColor</oldName>
```

Comment:
Since this node is being renamed for the second time, will two migration entries for the same node work?